src/user-management/user-management.ts

```diff
@@ -501,4 +536,12 @@ export class UserManagement {
 
     return `${this.workos.baseURL}/user_management/authorize?${query}`;
   }
+
+  getLogoutUrl({ sessionId }: { sessionId: string }): string {
+    if (!sessionId) {
+      throw new Error(`Incomplete arguments. Need to specify 'sessionId'.`);
```

Comment:
Should we throw a `TypeError` here instead?

I think it's generally the type used for errors in the way a function is called. ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/More_arguments_needed))

Reply:
ah good call. i've changed this (and the error we were throwing in getAuthorizationUrl (which i copied from). i think that's probably safe to do? if someone was doing an instanceof check on the exception before that should still pass.

Reply:
> if someone was doing an instanceof check on the exception before that should still pass.

That's right! Both are instances of `Error`.